katello/db_config.rb

```diff
@@ -0,0 +1,49 @@
+require 'yaml'
+
+module KatelloUtilities
+  class DBConfig
+    attr_reader :foreman, :candlepin
+    INSTALLER_CONFIG = '/etc/foreman-installer/scenarios.d/last_scenario.yaml'
+
+    def initialize(scenario = INSTALLER_CONFIG)
+      config = YAML.load(File.read(scenario))
+      answers = YAML.load(File.read(config[:answer_file]))
+      @foreman = {
+          :username => answers['foreman']['db_username'],
+          :password => answers['foreman']['db_password'],
+          :database => answers['foreman']['db_database'] || 'foreman',
+          :host => answers['foreman']['db_host'] || 'localhost',
+          :port => answers['foreman']['db_port'] || '5432'
+      }
+      @candlepin = {
+          :username => answers['katello']['candlepin_db_user'],
+          :password => answers['katello']['candlepin_db_password'],
+          :database => answers['katello']['candlepin_db_name'] || 'candlepin',
+          :host => answers['katello']['candlepin_db_host'] || 'localhost',
+          :port => answers['katello']['candlepin_db_port'] || '5432',
+          :ssl => answers['katello']['candlepin_ssl'] || false,
+          :ssl_verify => answers['katello']['candlepin_ssl_verify'] || false
```

Comment:
These 2 SSL options don't seem to actually be used?

Reply:
Currently not, but as it is part of the DB configuration I chose to keep it. It may become useful when building URI for JDBC like in https://github.com/Katello/katello-installer/blob/master/hooks/pre/30-upgrade.rb#L39. If it is a problem, I can remove it.

Reply:
Please do, we can add it when we need it. Seem unnecessary to me :)